File: Taller Tipos de clases/C#/Clase Normal/Producto.cs
class Producto
{
    public string Nombre;
    public double Precio;

    public Producto()
    {
        Nombre = "Undefined";
        Precio = 0.0;
    }

    public Producto(string nombre, double precio)
    {
        Nombre = nombre;
        Precio = precio;
    }

    public Producto(string nombre)
    {
        Nombre = nombre;
        Precio = 10.0;
    }

    public Producto(Producto p)
    {
        Nombre = p.Nombre;
        Precio = p.Precio;
    }
}

File: Taller Tipos de clases/JAVA/Clase Normal/Producto.java
class Producto {
    String nombre;
    double precio;

    public Producto() {
        nombre = "Undefined";
        precio = 0.0;
    }

    public Producto(String nombre, double precio) {
        this.nombre = nombre;
        this.precio = precio;
    }

    public Producto(String nombre) {
        this.nombre = nombre;
        this.precio = 10.0;
    }

    public Producto(Producto p) {
        this.nombre = p.nombre;
        this.precio = p.precio;
    }
}
